lib/ObisNames.js

```diff
@@ -363,89 +363,95 @@ const obisMeasurement = {
             98: 'List'
         },
         'de': {
-        	1: 'Kosten',
+            1: 'Kosten',
             70: 'Beschaffenheit',
-        	99: 'Profilwert',
+            99: 'Profilwert',
             96: 'Wartungseintrag',
             97: 'Fehler',
             98: 'Listen'
         },
         "nl": {
-            1: "kosten",
-            70: "textuur",
+            1: "Kosten",
+            70: "Toestand",
             99: "Profielwaarde",
             96: "Onderhoudsgegevens",
-            97: "Fout",
-            98: "Lijsten"
+            97: "Foutmelding",
+            98: "Lijst"
         }
     },
     5: { // Cooling
         'en': {
-            1: 'Cooling',
-            99: 'Cooling',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List'
+            1: "Cooling",
+            99: "Cooling",
+            96: "Service entry",
+            97: "Error message",
+            98: "List"
         },
         'de': {
-        	1: 'Kälte',
-        	99: 'Kälte',
-            96: 'Wartungseintrag',
-            97: 'Fehler',
-            98: 'Listen'
+            1: "Kälte",
+            99: "Kälte",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen"
         },
         "nl": {
-            1: "koud",
-            99: 'Cold',
+            1: "Koeling",
+            99: "Koeling",
             96: "Onderhoudsgegevens",
-            97: "Fout",
-            98: "Lijsten"
+            97: "Foutmelding",
+            98: "Lijst"
         }
     },
     6: { // Heating
         'en': {
-            1:  'Heat',
-            3:  'Heat',
-            99: 'Heat',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List'
+            1:  "Heat",
+            3:  "Heat",
+            99: "Heat",
+            96: "Service entry",
+            97: "Error message",
+            98: "List"
         },
         'de': {
-            1: 'Wärme',
-            3: 'Wärme',
-        	99: 'Wärme'
+            1: "Wärme",
+            3: "Wärme",
+            99: "Wärme"
+	    96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen"
         },
         "nl": {
-            1: "warmte",
-            3: 'Heat',
-            99: "Heat"
+            1: "Warmte",
+            3: "Warmte",
+            99: "Warmte"
```

Comment:
discutable, ik ken deze niet hebben denk ik te maken met een warmte pomp of Central warmte afname ?

Reply:
Weet ik ook niet